cluster/storage-cluster.go

```diff
@@ -0,0 +1,250 @@
+// This file is part of MinIO Kubernetes Cloud
+// Copyright (c) 2019 MinIO, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+package cluster
+
+import (
+	"errors"
+	"fmt"
+)
+
+type StorageCluster struct {
+	Id   int32
+	Name *string
+}
+
+type StorageClusterResult struct {
+	*StorageCluster
+	Error error
+}
+
+type AddStorageClusterResult struct {
```

Comment:
What is a `AddStorageClusterResult`? - Also why do we need a different type compared to `StorageClusterResult` that has no other fields. If really necessary maybe `type AddStorageClusterResult StorageClusterResult`

Reply:
my mistake xD didn't notice I had the same struct duplicate hahaha I'm glad we have code reviews.